### ISSUE TITLE
Passive event listeners support

### ISSUE THREAD

**Issue by fasterthanlime** [clauderic/react-sortable-hoc#210]
Admittedly I don't know much about how react-sortable-hoc works, but I'm seeing these
violations in my Chrome dev tools:

```
[Violation] Added non-passive event listener to a scroll-blocking 'touchstart' event. Consider marking event handler as 'passive' to make the page more responsive.
```

From the looks of https://stackoverflow.com/questions/39152877/consider-marking-event-handler-as-passive-to-make-the-page-more-responsive/39187679 it might be something react-sortable-hoc wants to support (trivial code change, too?)

Just wanted to make sure this was on your radar, feel free to close :)

**Comment by Holi0317** [comment]
Unfortunately, this library does use `event.preventDefault()` internally. See [this](https://github.com/clauderic/react-sortable-hoc/blob/ae50bb9ecd923c78b116064d38b392bef5557ece/src/SortableContainer/index.js#L356).

I tried to enable passive event listener on my build, but Chrome will spam out error message as I violated the passive event rule. 

Explicitly stating that is not a passive event still produce the violation message on dev tools. So we got no choice but to leave it as is.
